config/initializers/paperclip.rb

```diff
@@ -0,0 +1,17 @@
+Paperclip::Attachment.default_options.merge!(
+  path: ':rails_root/uploads/:class/:attachment/:id_partition/:style/:filename',
+  url: '/uploads/:class/:id/:style'
```

Comment:
I don't think the URL thing is right. I guess we can change later. Ultimately I'd like /en/m/foobar/media/6352. This comes up in issue 3908.